reinforcement_learning/rl_game_server_autopilot/sagemaker/rl_gamerserver_ray.ipynb

```diff
@@ -869,7 +869,8 @@
     "      ContentType=content_type,\n",
     "      Accept=accept,\n",
     "      Body=last_observations\n",
-    "    )"
+    "    )",
```

Comment:
Minor, should probably have trailing newline so that it doesn't look squished:
```
   "    )\n",
```

Reply:
Fixed. Thanks!